include/ideep/operators/batchnorm.hpp

```diff
@@ -192,6 +192,10 @@ struct batch_normalization_backward
           batch_normalization_flag::use_scale |
           batch_normalization_flag::use_shift,
       const engine& aengine = engine::cpu_engine()) {
+    IDEEP_CHECK(!(check_isa_avx2_vnni_2() &&
+                  utils::one_of(src.get_data_type(),
+                                data_type::bf16, data_type::f16)),
+                  "DNNL does not support bf16/f16 backward on the platform with avx2_vnni_2");
```

Comment:
Is it temporary due to onednn implementation?

Reply:
It is not temporary since oneDNN will not support bf16/fp16 backward on SRF.